src/System.Management.Automation/engine/remoting/commands/CustomShellCommands.cs

```diff
@@ -3971,8 +3924,7 @@ private PSObject ConstructPropertiesForUpdate()
                 {
                     using (System.Management.Automation.PowerShell ps = System.Management.Automation.PowerShell.Create())
                     {
-                        bool isWorkflowConfiguration = IsWorkflowConfigurationType(ps);
-                        if (!string.IsNullOrEmpty(modulePathParameter) && isWorkflowConfiguration)
+                        if (!string.IsNullOrEmpty(modulePathParameter))
                         {
                             List<object> modifiedModulePath = new List<object>(modulesToImport);
                             modifiedModulePath.Insert(0, ConfigurationDataFromXML.PSWORKFLOWMODULE);
```

Comment:
I think this entire conditional block can be removed.  We will never import a PSWORKFLOWMODULE as part of an endpoint configuration, since it is no longer supported in PSCore6.

Reply:
Agree. Done.